content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -8,7 +8,37 @@ menu:
     parent: About the project
 ---
 
+## v1.9.3 [2022-02-01]
+
+{{% note %}} To address a bug that impacted communication between the browser’s main thread and background workers, we bumped the 1.9.2 release version to 1.9.3.
+{{% /note %}}
+
+### Features
+- Add ability to rename TICKscripts.
+- Add the following enhancements to the `InfluxDB Query Management` page:
```

Comment:
` InfluxDB Admin - Queries tab`  I believe this is the exact thing where the changes were made.

Reply:
changed to InfluxDB Admin - Queries tab